### DISCUSSION TITLE
DLL

### DISCUSSION THREAD

**Discussion by Spokplacenta** [dotnet/MQTTnet#1308] [Q&A] [answered]
Hi,
Is it possible to have a MQTTnet.DLL to import in a project ?

Anthony

**Comment by Spokplacenta** [comment, marked as answer]
Ok i find it.
Nuget is a ZIP archive containing the DLL.

**Reply by SeppPenner** [reply]
> Ok i find it. Nuget is a ZIP archive containing the DLL.

I was about going to tell the same.
